### ISSUE TITLE
User for sudo changed after update and installation of Log-Viewer with openhabian-config

### ISSUE THREAD

**Issue by blachner** [openhab/openhabian#1752]
## Issue information:
- I had a working openhabian installation with OpenHAB 2.5
- I updated it via openhabian-config to OpenHAB 3.4
- After upgrade, I also switched the branch in openhabian-config to release.
- OpenHAB 3.4 works after upgrade (More or less, some configuration issues).
- Because log viewer wasn't working I installed it again vie openhabian-config "20 | Optional Components" -> "21 Log Viewer".

After this, sudo was broken, because user and group was set to user/group frontail.

After this I restored the old system from my backup and tried it again. With the same result: After new installation of Log Viewer, sudo is broken.

Maybe others had a similar problem, see [Discussion](https://community.openhab.org/t/sudo-not-working-any-more/140093/2).

## Debug information:
Sorry, no further information, besides all programs in /usr/bin are set to user/group frontail. Especially sudo, which breaks the system:
`-rwxr-xr-x 1 frontail frontail 135376 Jan 23 2021 sudo`

## System information:
openhabian based on Buster.

**Comment by mstormi** [comment]
You have a starting point with a system you don't describe in detail. 
All you say is you are still on buster - which is not supported for openHABian.

You did "whatever" to upgrade (you don't explain or provide logs of ).

You don't tell how to reproduce the issue.

Now your system broke and you open an issue for this although for sure isn't a general openHABian issue as you would not have been the first one for this to happen. So it's not. It's related to *your* system alone.

Github is a *development* tracking system, not a user helpline.
If you want to help advance OH and OH, put up a more useful post that properly takes the aforementioned into account.

**Comment by blachner** [comment]
The start system was openhabian. A version based on buster maybe from year 2019, with all updates till January 2023.
Also the openhabian-config script was the actual from your legacy branch.
I upgrade on this system with openhabian-config to the actual openHAB 3.
I switched to release branch in openhabian-config and updated the script.
I installed the Log viewer -> sudo and user programs in /usr/bin changed user/group to logtails.

I just don't know, which logs can help you here.

Yes, I used a ssd, but I don't think, that changed user and groups settings in any way are related, if the system is on ssd or sd.

For sure, install the logger with openhabian-config on some updated openhabian system, makes the system useless, because it breaks sudo. 

I can solve the problem for me by myself. Fresh install of openhabien bullseye and import the converted and backuped openhab 3 config from the updated system. On this system also no issues with installing log viewer.

I just opened this issue, because others maybe also affected by this issue.

But, if you think its not worth to search the bug in openhabian-config , its your decision.

**Comment by mstormi** [comment]
The point is it you ignored several prerequisites and ran a totally unsupported system and THAT is why you had that trouble.
It does not affect anyone but you so it isn't an openhabian bug so not worth looking after unless you can reproduce it on a properly setup, up to date installation of openHABian.

**Comment by blachner** [comment]
It was a normal openhabian installation. That this is probably end of life, I did not know, but is of course OK.

Maybe openhabian-config should indicate that this version is no longer supported, especially the update to openhab 3. I may simply not have seen the clues either.

In the forum discussion you see, that it  seems, that it affected at least one other person. For me, destroy sudo is a critical bug. I can reproduce it use upgrade from an old, now unsupported, openhabian version.  But fortunately I can't reproduce it with a fresh installation of an actual openhabian. 

But as I said, I solved the problems, so for me its OK. I was going to switch to bullseye anyway. And if it's your estimation that it's not worth troubleshooting here, that's perfectly fine.
